### PR TITLE
feature(ui-ux): add footer 

### PR DESCRIPTION
**Pull request by veralygit**
#### What this PR does / why we need it:
This PR is a PR to add Footer feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
gradient background position is a little off for md as it follows the Figma px spacing
The position of the footer and background is dependent on the components of the webpage

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

**Comment by kyleleow** [comment]
image
There is this empty space below footer when i scroll
- when my viewport is 1832x1304, 
- tablet view
- mobile view

File: src/pages/index.tsx [hunk 1]
```diff
@@ -11,8 +11,10 @@ export default function Home() {
       <SearchBar containerClass="px-5 md:px-10 lg:px-[316px] my-10" />
       <Container className="px-5 md:px-10">
         <TokenStatsDisplay />
-        <GroupChainStatsCard />
+        <GroupStatisticCard />
       </Container>
+      <div className="w-full h-screen absolute z-[-1] mix-blend-screen top-0 left-0 bg-no-repeat lg:bg-cover bg-contain lg:bg-[url('/background/gradient-purple.png')] md:bg-[url('/background/gradient-purple-tablet.png')] bg-[url('/background/gradient-purple-mobile.png')]" />
+      <div className="w-full h-screen absolute top-[210px] z-[-1] mix-blend-screen bg-no-repeat bg-contain bg-right sm:bg-[url('/background/gradient-blue-mobile.png')] bg-[url('/background/gradient-blue-mobile.png')]" />
```
Comment (kyleleow) [inline src/pages/index.tsx]: check we are using `/background/gradient-blue-mobile.png` for all viewports?

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      },
+    ],
+  },
+];
+
+function Footer() {
+  return (
+    <footer className="w-full relative pb-[40px] lg:pb-[70px] md:pb-[52px] sm:pb-[57.05] lg:pt-[104px] md:pt-[66px] pt-[79px] bottom-0 right-0">
+      <Container className="px-4 sm:px-10 md:px-10">
+        <div className="black-gradient-1-shadow backdrop-blur-[6px] bg-opacity-70 bg-black-900 flex flex-col sm:flex-row py-10 px-5 md:px-10 lg:px-[86px] rounded-[30px] black-gradient-1 gap-y-10 md:gap-0">
```
Comment (kyleleow) [inline src/components/Footer.tsx]: check vertical padding in mobile view

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+interface FooterColumnProps {
+  category: string;
+  childLinks: FooterLinkItemProps[];
+}
+
+interface FooterLinkItemProps {
+  label: string;
+  href?: string;
+  testId?: string;
```
Comment (kyleleow) [inline src/components/Footer.tsx]:
```suggestion
  testId: string;
```

Comment (veralygit) [reply src/components/Footer.tsx]: updated

File: src/components/Footer.tsx (hunk 2 above)
Comment (veralygit) [reply src/components/Footer.tsx]: updated mobile vertical margin between logo and links

File: src/pages/index.tsx (hunk 1 above)
Comment (veralygit) [reply src/pages/index.tsx]: removed BG in this PR will create a separate PR for BG

File: src/components/Footer.tsx (hunk 2 above)
Comment (kyleleow) [reply src/components/Footer.tsx]: image
i meant this

Comment (veralygit) [reply src/components/Footer.tsx]: updated 
image

**Comment by kyleleow** [comment]
image
to match horizontal padding in desktop view with figma

**Review by kyleleow — APPROVED** [review]

**Pull request merged**